ExtremelyBasic.java

```diff
@@ -0,0 +1,79 @@
+package week1;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.util.StringTokenizer;
+
+public class ExtremelyBasic {
+	static FastReader input = new FastReader();
+
+	public static void main(String[] args) {
+		System.out.println("X = " + (input.nextInt() + input.nextInt()));
+	}
+
+	static class FastReader {
+		BufferedReader br;
+		StringTokenizer st;
+
+		public FastReader() {
+			br = new BufferedReader(new InputStreamReader(System.in));
+		}
+
+		String next() {
+			while (st == null || !st.hasMoreTokens()) {
+				try {
+					st = new StringTokenizer(br.readLine());
+				} catch (IOException e) {
+					e.printStackTrace();
+				}
+			}
+			return st.nextToken();
+		}
+
+		int nextInt() {
+			return Integer.parseInt(next());
+		}
+
+		long nextLong() {
+			return Long.parseLong(next());
+		}
+
+		double nextDouble() {
+			return Double.parseDouble(next());
+		}
+
+		String nextLine() {
+			String str = "";
+			try {
+				str = br.readLine().trim();
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+			return str;
+		}
+	}
+
+	class FastWriter {
+		private final BufferedWriter bw;
+
+		public FastWriter() {
+			this.bw = new BufferedWriter(new OutputStreamWriter(System.out));
+		}
+
+		public void print(Object object) throws IOException {
+			bw.append("" + object);
+		}
+
+		public void println(Object object) throws IOException {
+			print(object);
+			bw.append("\n");
+		}
+
+		public void close() throws IOException {
+			bw.close();
+		}
+	}
+}
```

Comment:
why you are using FastReader  and FastWriter ?
The required Task is more **simple** just print summation of First number and second number.
please refactor the code